### PR TITLE
Toolboxes & welders update

### PR DESCRIPTION
**Pull request by ACCount12**
- Emergency welding tool added to emergency toolbox. It holds 2 times less fuel than regular welder.
- Changed industrial welding tool sprite to a cooler one.
- Replaced welder in syndicate toolbox with industrial one.
- All tools in syndicate toolbox are red. Corporate style!
- Red crowbar is a bit more robust (5 -> 8 force).
- Fixed two bugs: welder icon disappearing and drone toolbox spawning with wrong color cable coil.
- Changelog.

Welders:
welders

### CODE REVIEW

**Comment by Xhuis** [comment]
Could you post screenshots of the sprites?

**Comment by ACCount12** [comment]
Added screenshot.

**Comment by pudl** [comment]
";SECURITY TO BAR CHEF HAS A RED CROWBAR"

**Comment by Xhuis** [comment]
:+1:

**Comment by ACCount12** [comment]
Every tool in syndie toolbox can be found on station. No meta for you.

**Comment by Chocobro** [comment]
finally the industrial welder gets its own sprite :+1:

**Comment by pudl** [comment]
add the gauge from normal welders to the industrial and mini one

**Comment by ACCount12** [comment]
1. Your gauge sucks.
2. Industrial has egun-like line gauge on top.
3. Mini has no gauge, no handle, no safeties, no anything that is not needed for welding. Nanotrasen are cheap bastards.

**Comment by pudl** [comment]
>Nanotrasen are cheap bastards.
kill yourself

**Comment by GunHog** [comment]
I agree, that normal gauge for current welders is too small and difficult to read. I am loving the industrial welder :+1:

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -425,9 +443,9 @@
 	return (BRUTELOSS)
 
 /obj/item/weapon/crowbar/red
-	icon = 'icons/obj/items.dmi'
 	icon_state = "red_crowbar"
 	item_state = "crowbar_red"
+	force = 8
```
Comment (Jordie0608) [inline code/game/objects/items/weapons/tools.dm]: I don't object exactly, but why?

**Comment by JJRcop** [comment]
If the emergency welder really is from cheap bastards, why should it be able to turn off?

**Pull request merged**